projects/examples/src/components/quick-search/filters/quick-search-filters.example.component.ts

```diff
@@ -0,0 +1,174 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import {
+    QuickSearchFilter,
+    QuickSearchProviderDefaults,
+    QuickSearchRegistrarService,
+    QuickSearchResultItem,
+    QuickSearchResultsType,
+} from '@vcd/ui-components';
+import Mousetrap from 'mousetrap';
+
+@Component({
+    selector: 'vcd-quick-search-sync-async-example',
+    styleUrls: ['./quick-search-filters.example.component.scss'],
+    templateUrl: './quick-search-filters.example.component.html',
+    providers: [QuickSearchRegistrarService],
+})
+export class QuickSearchFiltersExampleComponent implements OnInit, OnDestroy {
+    formGroup: FormGroup;
+    kbdShortcut = 'mod+f';
+    spotlightOpen: boolean;
+
+    filters: QuickSearchFilter[] = [
+        {
+            id: 'type',
+            options: [{ display: 'action' }, { display: 'other' }],
+        },
+    ];
+
+    private lazyLoadedProvider = new LazyLoadedActionsSearchProvider();
+    private actionsSearchProvider = new ActionsSearchProvider();
+
+    private readonly mousetrap: MousetrapInstance;
+    constructor(private fb: FormBuilder, private searchRegistrar: QuickSearchRegistrarService) {
+        // Create an instance of mouse trap within the constructor so that any bount shortcut event handler
+        // would be executed within the angular zone
+        this.mousetrap = new Mousetrap();
```

Comment:
Do we really need mouse trap? We should avoid libraries in the examples, they should  be as simple as possible. It was a mistake to use it before.

It's very simple to set a key handler with a modifier with angular

```<input (keydown.meta.f)="onKeydown($event)">```

Reply:
Removed mousetrap, it opens with the button now (which is much larger in this example)